### PR TITLE
[CAL-1095] Embed modal (floating pop up button) - UI/layout/spacing issues

### LINKED ISSUE #7156: [CAL-1095] Embed modal (floating pop up button) - UI/layout/spacing issues
Currently

Should be

[View in Figma](https://www.figma.com/file/xk4HOxtSI82J0F7enMxeak/Cal---Live?node-id=22%3A38088&t=AQX9GWSFsCzlRrEy-1)

Note: No need to touch the preview - just the modal and left panel UI.

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-1095](https://linear.app/calcom/issue/CAL-1095/embed-modal-floating-pop-up-button-uilayoutspacing-issues)</sub>

### PR DESCRIPTION
**Pull request by gitstart-calcom**
Fixes #7156

## Demo
https://www.loom.com/share/6f52f4e1fab640e39dd0ec5717ace0d0

### CODE REVIEW

File: packages/ui/components/form/color-picker/colorpicker.tsx [hunk 1]
```diff
@@ -2,13 +2,15 @@ import * as Popover from "@radix-ui/react-popover";
...
 
+import cx from "@calcom/lib/classNames";
 import { fallBackHex, isValidHexCode } from "@calcom/lib/getBrandColours";
 
 export type ColorPickerProps = {
   defaultValue: string;
   onChange: (text: string) => void;
   container?: HTMLElement;
   popoverAlign?: React.ComponentProps<typeof Popover.Content>["align"];
+  colorInputClassName?: string;
```
Comment (CarinaWolli) [inline packages/ui/components/form/color-picker/colorpicker.tsx:13]:
```suggestion
  className?: string;
```

**Review by CarinaWolli — DISMISSED** [review]
Tablet view is broken, let's just put _button color_ and _text color_ under each other when the screen is getting too small: 
Screenshot 2023-04-13 at 10 14 36

Everything else looks good to me 🙏

**Comment by PeerRich** [comment]
we should also extract all strings into en/common.json

**Comment by hariombalhara** [comment]
Screenshot 2023-04-25 at 4 15 07 PM
Screenshot 2023-04-25 at 4 15 18 PM

Too much spacing above Theme Selector which I think is unexpected.

**Review by hariombalhara — CHANGES_REQUESTED** [review]
Left a comment. Apart from that it's good.

**Review by hariombalhara — APPROVED** [review]

**Comment by hariombalhara** [comment]
@CarinaWolli I guess it just needs your review to be merged.

**Review by keithwillcode — APPROVED** [review]

**Pull request merged**